app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -66,12 +66,17 @@ def create_submission(saved_claim)
       Rails.logger.info(
         'Creating 526 submission', user_uuid: @current_user&.uuid, saved_claim_id: saved_claim&.id
       )
-      Form526Submission.create(
-        user_uuid: @current_user.uuid,
-        saved_claim_id: saved_claim.id,
-        auth_headers_json: auth_headers.to_json,
-        form_json: saved_claim.to_submission_data(@current_user)
-      )
+      ActiveRecord::Base.transaction do
```

Comment:
not that it's your doing, but the whole `create_submission` method seems like it belongs in the Form526Submission model.